webpack.config.cjs

```diff
@@ -218,6 +219,22 @@ const webStoriesScripts = {
   ].filter(Boolean),
 };
 
+// Collect all core themes style sheet paths.
+const coreThemesBlockStylesPaths = glob.sync(
+  './assets/src/web-stories-block/css/core-themes/*.css'
+);
+
+// Build entry object for the Core Themes Styles.
+const coreThemeBlockStyles = coreThemesBlockStylesPaths.reduce((acc, curr) => {
+  const length = curr.indexOf('.css') - curr.indexOf('core-themes/');
+  const fileName = curr.substr(curr.indexOf('core-themes/'), length);
```

Comment:
Could just do `import { basename }  from 'path'` and then `const fileName = basename(curr);`